src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  background-color: #fff;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+}
+
+.moyo__logo {
+  margin-top: 5px;
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). Style all elements using '.class'. The '.moyo__logo' class should include the margin-top style instead of applying it directly to the HTML tag.